std-bits/table/src/main/java/org/enso/table/data/column/storage/numeric/LongConstantStorage.java

```diff
@@ -0,0 +1,15 @@
+package org.enso.table.data.column.storage.numeric;
+
+public class LongConstantStorage extends ComputedLongStorage {
+  private final long constant;
+
+  public LongConstantStorage(long constant, int size) {
+    super(size);
+    this.constant = constant;
+  }
+
+  @Override
+  protected long computeItem(int idx) {
+    return constant;
+  }
+}
```

Comment:
This class, together with `ComputedLongStorage` demonstrates how we can implement O(1) storages for constant columns.

At some point we may do a similar refactor for all other storage types (we should be able to share most of the code for the non-primitive storages, only DoubleStorage will also need special handling to avoid unnecessary boxing, like we are doing here) to support the constant columns better.